lib/active_meta/attribute.rb

```diff
@@ -31,9 +31,7 @@ def overload(&block)
     end
 
     def register_rule(rule)
-      unless rule.is_a? ActiveMeta::Rule
-        raise ArgumentError, "no rule given for attribute #{@attribute}"
-      end
+      raise ArgumentError, "no rule given for attribute #{@attribute}" unless rule.is_a? ActiveMeta::Rule
```

Comment:
Good idea ! Personally, I refer to the [Github's one](https://github.com/styleguide/ruby).